lib/wit.rb

```diff
@@ -59,6 +59,24 @@ def interactive(handle_message=nil, context={})
     puts
   end
 
+  def create_new_app(payload, set_new_app_token = false)
+    response = req(logger, @access_token, Net::HTTP::Post, "/apps", {}, payload)
+    @access_token = response['access_token'] if set_new_app_token
+    return response
+  end
+
+  def train(payload)
+    payload = [payload] if payload.is_a? Hash
+
+    payload.each do |utternace|
```

Comment:
```suggestion
    payload.each do |utterance|
```

Reply:
i can't get what change do you suggest here?